src/Tools/onnx-mlir-opt/onnx-mlir-opt.cpp

```diff
@@ -108,6 +108,14 @@ void scanAndSetMAccel(int argc, char **argv) {
   }
 }
 
+std::string getVendorName() {
```

Comment:
@mikeessen This looks good to me but I noticed you had to define ` getVendorName()` in multiple files.... I wonder if there is a centralized location where we could define ` getVendorName()` only once and call it from the other files that also need it?  Since I am still new to the `onnx-mlir` repo I will defer to @AlexandreEichenberger whether that is a reality or not but either way looks good.

Reply:
yes, that is a good suggestion @hamptonm1. You can create a new file(s), and lib if necessary.

Reply:
(or maybe you have 2 copies because one is for "onnx-mlir/XXX" and the other one is for "onnx-mlir-opt/XXX-opt")

Reply:
@AlexandreEichenberger @hamptonm1 
Thanks for the suggestions.

I don't see a need for two versions of getVendorName() as the same vendor name is used for onnx-mlir and onnx-mlir-opt when PACKAGE_VENDOR is specified for make processing and the same default vendor name is used when not specified.  

 I created one version of getVendorName() in the existing CompilerUtils.  Please take a look.

Reply:
@mikeessen This update looks good....you would just need to update your branch with the latest changes from main and make sure all checks pass :)